### DISCUSSION TITLE
Running Sloeber on M1/M2 Mac on ARM64 natively

### DISCUSSION THREAD

**Discussion by robzed** [Sloeber/arduino-eclipse-plugin#1560] [Q&A]
Running Sloeber 4.4 on an M2 Mac - seems to run fine via Rosetta/Intel. 
I'm also running 2023 Eclipse which runs native on Apple ARM. 
I wondered what was the easiest way to convince Sloeber to run via Apple ARM - I assume it's via updated JRE?

**Comment by jantje** [comment]
I do not own a apple so I can't help you.
Hopefully someone else can.

**Reply by robzed** [reply]
Which version of Eclipse does Version 4.4.1 contain? (I think 4.4 is v4.18).

They've had macOS AArch64 version since 2021-12 (4.22).

I'm also running Eclipse 2023-03 (4.27) which is native. Will the Sloeber plug-in run on this version?

**Reply by jantje** [reply]
I'm not sure what you are asking but sloeber has been running native on mac os for a very long time.
Sloeber always builds with the latest released eclipse. 4.4.4 has been build on 5 September 2022 so that should be eclipse 2022-09 (or 06)
> Will the Sloeber plug-in run on this version?

There is only one way to find out: try

**Reply by robzed** [reply]
Tried adding the Sloeber plug-in via the Eclipse Marketplace ...

An error occurred while collecting items to be installed
session context was:(profile=_Users_rob_eclipse_embedcpp-2023-03_Eclipse.app_Contents_Eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
No repository found containing: osgi.bundle,org.eclipse.nebula.cwt,1.1.0.202303072132
No repository found containing: osgi.bundle,org.eclipse.nebula.widgets.cdatetime,1.5.0.202303072132
No repository found containing: osgi.bundle,org.eclipse.nebula.widgets.oscilloscope,1.4.0.202303072132
No repository found containing: org.eclipse.update.feature,org.eclipse.nebula.widgets.oscilloscope.feature,1.4.0.202303072132
No repository found containing: osgi.bundle,org.eclipse.nebula.widgets.oscilloscope.source,1.4.0.202303072132
No repository found containing: osgi.bundle,org.junit,4.13.2.v20211018-1956

**Reply by jantje** [reply]
https://github.com/Sloeber/arduino-eclipse-plugin/issues/1558
something with https and http for nebula

**Reply by robzed** [reply]
hmmm... I'll take a look at it tomorrow.
